README.md

```diff
@@ -657,7 +657,7 @@ In case you want to monitor all namespaces in a cluster, you can add the followi
 [embedmd]:# (examples/all-namespaces.jsonnet)
 ```jsonnet
 local kp = (import 'kube-prometheus/kube-prometheus.libsonnet') +
-  (import 'kube-prometheus/kube-prometheus-all-namespaces.libsonnet') + {
+           (import 'kube-prometheus/kube-prometheus-all-namespaces.libsonnet') + {
```

Comment:
out of curiosity: is this a result of `jsonnetfmt`ing?

Reply:
IIUC the CI only `make generate`, not `make generate fmt`. Maybe it should? cc @paulfantom

Reply:
Yes, this is a result of `jsonnetfmt`.

@simonpasquier Maybe you run `make` at some point? This includes `fmt` target :shrug:

Reply:
yep I think I executed `make`